### DISCUSSION TITLE
How to transfer legacy workspace to organizations?

### DISCUSSION THREAD

**Discussion by osseonews** [nhost/nhost#3074] [Q&A]
There was recently a change to deprecate workspaces to organizations. When we log into our paid account with Nhost, it says we are on a legacy plan. In your post about this [change](https://nhost.io/blog/organization-billing)  you say: `"However, for paid projects, you will need to handle the migration yourself. Don’t worry — this process will be straightforward, requiring just a couple of clicks, and you’ll have plenty of time to complete it."`

Unfortunately, this process is not in the least bit straightforward. There is absolutely no instructions whatsoever on the dashboard on what to do and obviously we don't want to mess up our set up in anyway. How do we do the migration? Where are the step by step instructions? Incidentally, I apologize if I am missing something, but your post about this is very disturbing. You are migrating free customers who don't pay you a penny and also helping Team and Enteprise, but paying Pro customers can do it themselves? Your Pro Customers are unimportant to you? That's what it sounds like in your post. Your Pro customers are the feeding ground for Team and Enterprise.  It is entirely inappropriate to expect paying users, who are not Team or Enterprise to migrate themselves without any help, especially when free customers are migrated by you also.

**Comment by dbarrosop** [comment]
> Unfortunately, this process is not in the least bit straightforward. There is absolutely no instructions

There should be a transfer button in your projects in legacy workspaces with instructions.

> You are migrating free customers who don't pay you a penny and also helping Team and Enteprise, but paying Pro customers can do it themselves? Your Pro Customers are unimportant to you? That's what it sounds like in your post

Not sure why you are inferring that. We are migrating free customers because there is nothing to migrate, just a change in our internal data. Pro customers need to migrate their projects to an organization of their choosing because we don't know how many organizations you may want (i.e. do you want a single org for all your projects, one org per project, one org with projects X and Y and another one with project Z?). And teams and enterprise are paying for support so that's what they are getting :)

**Comment by osseonews** [comment]
Sorry, I don't see any button or instructions anywhere in our account. And this is not a support issue that warrants an enterprise account. That is ludicrous. the change is not coming from the customer end. you made the changes and effected all your customers accounts. you are responsible for helping solve the situation for all paying customers.

**Reply by dbarrosop** [reply]
> I don't see any button or instructions anywhere in our account

If you don't see the transfer button in the overview page of your project, please, open a support ticket.

> And this is not a support issue that warrants an enterprise account

Nobody said it was. You can open a support ticket and the support team will gladly help you like it has helped many others.

**Reply by osseonews** [reply]
OK, emailing support. BTW, you are trying to increase revenue from existing customers. Organizational billing will help you do that. It's a great change. But you are making it complicated. You should migrate all customers and not make it a pain to do so. I shouldn't have to come on Github an open an issue to try to increase my business with you and fill out some support form to migrate to a new paying structure, that your company changed. Now I don't even think I understand how organizational pricing even works based on your questions. Anyway, will close this issue and hopefully figure it out with support email. Thanks.

**Comment by osseonews** [comment]
BTW, finally found the button after clicking around. But, then you have a form to fill out to create a new organization, if you want to migrate. When you try to create a new organization, you get the error below. 

```
"error": {
    "name": "ApolloError",
    "graphQLErrors": [
      {
        "message": "invalid redirect URL",
        "path": [
          "billingCreateOrganizationRequest"
        ]
      }
    ],
    "protocolErrors": [],
    "clientErrors": [],
    "networkError": null,
    "message": "invalid redirect URL",
    "cause": {
      "message": "invalid redirect URL",
      "path": [
        "billingCreateOrganizationRequest"
      ]
    }
  }
}
```

**Reply by dbarrosop** [reply]
Thanks for the report, looking at it right now.

**Reply by dbarrosop** [reply]
This should work again. Thanks again, we just did some internal migration and looks like we didn't update this parameter correctly.
